plugins/zoom/icon_zoom/tasks/monitor_sign_in_out_activity/task.py

```diff
@@ -116,17 +120,31 @@ def loop(self, state: Dict[str, Any], custom_config: Dict[str, Any]):  # noqa: C
                         cutoff_date.get("second", 0),
                     )
                 )
-            else:
+            elif cutoff_hours is not None:
+                self.logger.info("Setting min start time (cutoff hours manually applied)")
```

Comment:
for checking the right values have been passed from komand-props should this logger include the `cutoff_hours` or add a similar logger within the `get_datetime_last_x_hours ` to log how many hours back are being calculated and then it would work for the next condiitons on state vs non state in the next conditon block? 

also minor we could then just pass this variable into `get_datetime_last_x_hours`